pkgs/build-support/cc-wrapper/macos-sierra-reexport-hack.bash

```diff
@@ -2,67 +2,113 @@
 
 set -eu -o pipefail
 
+# For cmd | while; read; do ...; done
```

Comment:
?

Reply:
This is needed for the block after the pipe (the `while` loop here) to set Bash variables. Otherwise it runs in a subshell and the variables it sets are not visible outside that block.

Reply:
I'm asking about the comment :)

Reply:
I wouldn't have though of this reason without the comment, so it was useful for me.

Reply:
Yeah it's not clear to my lastpipe is always an improvement, unlike the other extensions I used, so I wanted to motivate it.

I should remove the extra semicolon, however :D

Reply:
Removed!